crates/iota/src/client_ptb/ptb.rs

```diff
@@ -37,6 +38,12 @@ use crate::{
 pub struct PTB {
     #[arg(trailing_var_arg = true, allow_hyphen_values = true)]
     pub args: Vec<String>,
+    /// Select which fields of the response to display.
+    /// If not provided, all fields are displayed.
+    /// The fields are: effects, input, events, object_changes,
```

Comment:
I'm being annoying one last time, can we also revert effect/input in the comments/docs please?

Reply:
https://github.com/iotaledger/iota/pull/6274/commits/1959b4df31f8c6d9a2ed83e07cdfb8bc70396417